src/shared/utils/utils.js

```diff
@@ -0,0 +1,12 @@
+export const injectPropsIntoStepArray = (steps, stepId, props) => {
```

Comment:
What's this?

Reply:
This was something needed for the previous approach of defining an object with all the steps information. As it is no longer needed, I'll be deleting it.